kubernetes/eventhandler.go

```diff
@@ -209,16 +208,16 @@ func (*namespacePodUpdater) OnDelete(interface{}) {}
 type nodePodUpdater struct {
 	handler     podUpdaterHandlerFunc
 	store       podUpdaterStore
-	nodewatcher OldobjectWatcher
+	nodeWatcher oldObjectWatcher
```

Comment:
sorry I meant to say to replace any other reference to oldObject -> cachedObject

Reply:
Done @gsantoro 
thank for insisting on this

Also relevant updates done in tests of https://github.com/elastic/beats/pull/37431
Could you also have a look and approve there as well?